README.md

```diff
@@ -1,12 +1,23 @@
 # Concurrent Programming with Effect Handlers 
+<!-- [![Build Status](https://travis-ci.org/ocamllabs/ocaml-effects-tutorial.svg?branch=master)](https://travis-ci.org/ocamllabs/ocaml-effects-tutorial)    -->
+[![Build Status](https://github.com/tanelso2/ocaml-effects-tutorial/actions/workflows/prs.yml/badge.svg)](https://github.com/tanelso2/ocaml-effects-tutorial/actions/workflows/prs.yml)
 
-[![Build Status](https://travis-ci.org/ocamllabs/ocaml-effects-tutorial.svg?branch=master)](https://travis-ci.org/ocamllabs/ocaml-effects-tutorial)   
-
-Materials for the [CUFP 17 tutorial](http://cufp.org/2017/c3-daniel-hillerstrom-kc-concurrent-programming-with-effect-handlers.html).
+Originally written as materials for the [CUFP 17 tutorial](http://cufp.org/2017/c3-daniel-hillerstrom-kc-concurrent-programming-with-effect-handlers.html).
 
 ## Setting up
 
-Install the Multicore OCaml compiler [using opam](https://github.com/ocaml-multicore/multicore-opam#install-multicore-ocaml).
+Install a compatible OCaml compiler
+
+```bash
+opam switch create 5.0.0~beta1
+```
```

Comment:
This might get stale quickly with new releases. We're keeping the instructions here updated: https://github.com/ocaml-multicore/awesome-multicore-ocaml#installation. Maybe we could just add a reference to this for the installation instructions?

Reply:
Done